### ISSUE TITLE
Pulling from ad tables(ads, ads_insights, campaigns, adsets etc... ) getting "Cannot include video_p100_watched_actions...

### ISSUE THREAD

**Issue by johnpaulpg** [facebook/facebook-python-business-sdk#667]
# Facebook Python Business SDK Issue
Full error below, but we get a variety of errors showing different columns that is 'cannot include'. This is just started to happen recently.

400 Message: (#100) Cannot include 	video_p100_watched_actions, unique_inline_link_clicks, account_name, clicks, ad_id, reach, website_ctr, video_p50_watched_actions
						, cost_per_unique_click, canvas_avg_view_percent, engagement_rate_ranking, adset_name, quality_ranking, campaign_name, ctr, unique_link_clicks_ctr
						, cpm, outbound_clicks, video_30_sec_watched_actions, account_id, frequency, cpp, spend, inline_post_engagement, action_values, cost_per_unique_inline_link_click
						, impressions, unique_clicks, objective, unique_ctr, video_p25_watched_actions, ad_name, campaign_id, canvas_avg_view_time, conversion_rate_ranking
						, unique_inline_link_click_ctr, unique_actions, actions, cost_per_inline_link_click, cost_per_inline_post_engagement, unique_outbound_clicks
						, video_play_curve_actions, inline_link_click_ctr, inline_link_clicks, adset_id, cpc, video_p75_watched_actions
in summary param because they weren't there while creating the report run. All available values are: ".

Is this something we need to be concerned about? Are we losing any data because of this error?
